package.json

```diff
@@ -83,5 +83,7 @@
     "url": "https://github.com/opynfinance/GammaProtocol/issues"
   },
   "homepage": "https://github.com/opynfinance/GammaProtocol#readme",
-  "dependencies": {}
+  "dependencies": {
+    "ethers": "^5.0.7"
+  }
```

Comment:
Maybe be in devDependencies like any other packages?

Reply:
this is removed in other PR.